test/autoCompletion.test.ts

```diff
@@ -45,7 +45,9 @@ describe('Auto Completion Tests', () => {
     if (typeof position === 'undefined') {
       position = content.indexOf('|:|');
       content = content.replace('|:|', '');
+      if (content.length) position--;
```

Comment:
I made a calculation error in the initial PR. With the exacting checks I made (per the description of this PR), I corrected it here.